extensions/blocks/podcast-player/style.scss

```diff
@@ -100,30 +101,41 @@ $player-float-background: $light-gray-200;
 		height: $cover-image-size;
 	}
 
-	.jetpack-podcast-player__title {
-		display: flex;
-		flex-direction: column;
-		margin: 0;
-		overflow: hidden;
-	}
-
-	.jetpack-podcast-player__current-track-title {
-		font-size: $track-title-font-size;
-		margin: 0 0 $track-title-b-margin;
-	}
-
-	.jetpack-podcast-player__podcast-title {
-		font-size: $podcast-title-font-size;
-		color: $text-color;
-		margin: 0;
-	}
-
-	a.jetpack-podcast-player__podcast-title {
-		text-decoration: none;
-
-		&:hover,
-		&:focus {
-			color: $text-color-hover;
+	// Increasing specificity on the header titles and links for more consistency across themes and editor canvas.
+	.jetpack-podcast-player__header {
+		
+		.jetpack-podcast-player__title {
+			display: flex;
+			flex-direction: column;
+			margin: 0;
+			overflow: hidden;
+			letter-spacing: 0; // Fixes Twenty Twenty compressed text.
+			
+			&:before,
+			&:after {
+				display: none;
+			}
+		}
+	
+		.jetpack-podcast-player__current-track-title {
+			color: $player-title-color;
+			font-size: $track-title-font-size;
+			margin: 0 0 $track-title-b-margin;
+		}
+	
+		.jetpack-podcast-player__podcast-title {
+			font-size: $podcast-title-font-size;
+			color: $text-color;
+			margin: 0;
+		}
+	
+		.jetpack-podcast-player__podcast-title {
```

Comment:
Unless I'm missing something it looks like these two instances of `.jetpack-podcast-player__podcast-title` could be combined.

Reply:
Nice catch! Fixed by https://github.com/Automattic/jetpack/pull/15320/commits/9f16ef920c4b62fed756bdc5ed23cd86aefb87fd